### PR TITLE
COBRA-3701: dotenv

### PR DESCRIPTION
**Pull request by calaway**
Automatically read in environment variables from a `.env` file.

### CODE REVIEW

File: lib/config.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+require('dotenv').config();
```
[the remark this answers is not in the document]
Comment (trevorlinton) [reply lib/config.js:1]: how does dotenv work if we have multiple env files?

[the remark this answers is not in the document]
Comment (trevorlinton) [reply lib/config.js:1]: just to clarify, if there's no .env it does nothing correct?

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+NGROK_TOKEN=
+NODE_ENV=dev
+ONE_PROCESS_MODE=true
+PAPERTRAIL_DRAIN=
+PROMETHEUS_VERSION=v2
+SITE_BASE_DOMAIN=
+TEST_MODE=true
+TEST_REGION=us-seattle
+TWILIO_AUTH_KEY=
+US_SEATTLE_REGION_API=
```
[the remark this answers is not in the document]
Comment (trevorlinton) [reply .env.example:32]: Some of these are obviously test mode config vars but overall i think this is comprehensive, and nothing OC Tanner specific

**Review by trevorlinton — APPROVED** [review]

File: lib/config.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (calaway) [reply lib/config.js:1]: Correct, if there's no `.env`, as in production & CI environments, then nothing happens. It should only ever be used locally.

In fact, if we prefer we can add it to a `test` and `dev` script instead (e.g. `node -r dotenv/config your_script.js`) and that way we don't need to require it in the code at all. Let me know if you would prefer this and I can try it out.

[the remark this answers is not in the document]
Comment (calaway) [reply lib/config.js:1]: This package is really only made to work with one `.env` file. I've worked with dotenv libraries that will allow you to configure more than one env file, but this one is a bit more prescriptive and only accepts one.

[the remark this answers is not in the document]
Comment (trevorlinton) [reply lib/config.js:1]: I think it'll work for us though, we can have .env point to maru and make it explicit if you want to point it to another environment.

**Pull request merged**